metadata.yaml

```diff
@@ -1,31 +1,39 @@
-name: mongodb
+name: mongodb-k8s
 summary: A MongoDB operator charm for Kubernetes
 description: |
     MongoDB is a general purpose distributed document database.  This
     charm deploys and operates MongoDB on kubernetes Clusters. It
     supports replicated MongoDB databases.
-series:
-   - kubernetes
 tags:
   - kubernetes
   - mongodb
   - db
-min-juju-version: 2.7.5
-storage:
-  db:
-    type: filesystem
-    location: /data/db
-deployment:
-  type: stateful
-  service: cluster
+platforms:
+  - kubernetes
+bases:
+  - name: ubuntu
+    channel: 20.04/stable
```

Comment:
As far as I understand, this is not necessary

Reply:
Noted will remove.